domain/crossmodelrelation/state/model/remoteapplication.go

```diff
@@ -477,23 +476,20 @@ VALUES ($remoteApplicationConsumer.*);`
 	return nil
 }
 
-func (st *State) insertApplicationRemoteRelation(
+func (st *State) insertSyntheticRelation(
 	ctx context.Context,
 	tx *sqlair.TX,
 	consumerRelationUUID string,
-) (applicationRemoteRelation, error) {
-	// Create a relation UUID for the synthetic relation in the offerer model.
-	relationUUID, err := internaluuid.NewUUID()
-	if err != nil {
-		return applicationRemoteRelation{}, errors.Errorf("generating offer connection UUID: %w", err)
-	}
+) error {
 	relationID, err := sequencestate.NextValue(ctx, st, tx, domainrelation.SequenceNamespace)
 	if err != nil {
-		return applicationRemoteRelation{}, errors.Errorf("getting next relation id: %w", err)
+		return errors.Errorf("getting next relation id: %w", err)
 	}
 
+	// We use the same UUID for relations both in the consuming and offering
```

Comment:
Expand this a little. Say that the consuming side registers the relation using its own UUID, and we use the same UUID on the offering side.